### PR TITLE
Prepare release 0.4.1

### PR DESCRIPTION
**Pull request by mikhail**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,28 @@
+## Version 0.4.1
+ * #435: Pass Kingpin ECS actor ENV vars to task definition ([@diranged])
+ * #434: SYSTEMS-957: Support NoEcho Parameters in CloudFormation stacks. ([@diranged])
+ * #433: ECS: Fix a race condition in stop service ([@swaggy])
+ * #432: New Actor: spottinst.ElastiGroup ([@diranged])
+ * #431: Compare S3 bucket tags with some intelligence. ([@diranged])
+ * #429: Quiet down a logging statement ([@diranged])
+ * #428: Strip out unicode characters in diff_dicts. ([@swaggy])
+ * #427: Bugfix on S3 Bucket Tag comparison. ([@mikhail])
+ * #426: Add CAPABILITY_NAMED_IAM to the enumeration of capabilities parameters. ([@sabw8217])
```
Comment (diranged) [inline CHANGELOG.md]: add @sabw8217 to the bottom list of users

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,28 @@
...
+ * #412: Bugfix -- handling a single alertspec on a template was broken ([@diranged])
+ * #411: Bugfix: Make sure we set self.changed=True after updating the runnabl… ([@diranged])
+ * #410: Fix the RightScale ServerTemplate/Alert integration tests.. ([@diranged])
+ * #409: Delete .base.py.swo ([@diranged])
+ * #407: Bugfix - threshold must be a string. ([@diranged])
+ * #402: Create a new AlertSpecBase ensurable base actor for managing RightScale AlertSpecs ([@diranged])
+ * #393: Bugfix -- we couldn't create S3 buckets in anything other than the default region. ([@diranged])
+ * #391: Bugfix -- if no cloudformation parameters, thats fine. ([@diranged])
+ * #389: Remove bogus testing log line that was left there accidentally. ([@diranged])
+ * #385: Add ecs service actor. ([@swaggy])
```
Comment (diranged) [inline CHANGELOG.md]: @swaggy isnt a real member i think, double check the bottom list..

File: CHANGELOG.md (hunk 1 above)
Comment (mikhail) [inline CHANGELOG.md]: Done

File: CHANGELOG.md (hunk 2 above)
Comment (mikhail) [inline CHANGELOG.md]: He is :)

**Pull request merged**